static/sass/styles.scss

```diff
@@ -203,3 +203,9 @@ $color-navigation-background: #252525;
 
 @import 'snapcraft-publicise';
 @include snapcraft-publicise;
+
+html,
+body {
+  background: #fff; // This will be fix on vanilla with https://github.com/vanilla-framework/vanilla-framework/issues/2129
+  min-height: 100vh;
```

Comment:
Part about `min-height` on `html, body` should stay in sticky footer, because that's what it is for.

All we need for background is `body { background: $color-x-light }`.